Engine/Entities/Entity.cs

```diff
@@ -49,5 +50,14 @@ public Entity(string name, Transform transform, World world)
         /// Called by the world when this entity needs to update the status of each of its components.
         /// </summary>
         public abstract void Update();
+
+        /// <summary>
+        /// Displays a message in the console.
+        /// </summary>
+        /// <param name="msg">The message to display</param>
+        public virtual void Log(string msg)
+        {
+            Console.WriteLine($"[{GetType().Name}] {Name} : {msg}");
+        }
```

Comment:
Couldn't we abstract this a bit more? Maybe we could create a class that allows for logging & make entity inherit it?

Reply:
Other non-entity objects could want to log things in the console

Reply:
> We could think about that but I don't think abstracting more our program is useful lol

MOAR ABSTRACTION (or not, I agree with you)